package.json

```diff
@@ -6,7 +6,7 @@
   "author": "Ammar-64",
   "license": "MIT",
   "scripts": {
-    "start": "node src/app.js",
+    "start": "nodemon src/app.js",
```

Comment:
Since we're deploying the app to `Heroku`, we should use `node` not `nodemon`. 
So revert this change, please.

If you want to work on local with nodemon, you could just type `nodemon` at the command line, or `yarn watch`